src/evm/protocol/uniswap_v2/reserve_price.rs

```diff
@@ -1,18 +1,17 @@
-use ethers::types::U256;
-
 use crate::u256_num::u256_to_f64;
+use alloy_primitives::U256;
```

Comment:
Sorry if it's a stupid question, but in some places you've introduced BigUInt - and here, Alloy U256 - what's the reason? Capture overflows the same way it'd happen on-chain?

Reply:
If I understand correctly, Alloy for EVM-related code and BigUint for Tycho types

Reply:
yes exactly! BigUInt for interfaces and alloy for EVM related code! (Adding this in the PR description as well)